src/wiktextract/extractor/simple/page.py

```diff
@@ -0,0 +1,149 @@
+    # https://simple.wiktionary.org/wiki/Wiktionary:Entry_layout_explained
+
+    if page_title.startswith("Appendix:") or page_title in (
+        "Main Page",
+        "Main page",
+    ):
+        return []
+
+    if wxr.config.verbose:
+        logger.info(f"Parsing page: {page_title}")
+
+    wxr.config.word = page_title
+    wxr.wtp.start_page(page_title)
+
+    page_text = preprocess_text(page_text, page_title=page_title)
+
+    #####
+    # Temporary debug print stuff goes into debug_bypass
+
+    # from .debug_bypass import debug_bypass
+    # return debug_bypass(wxr, page_title, page_text)
+
+    #####
+
+    # Parse the page, pre-expanding those templates that are likely to
+    # influence parsing; these are problematic templates that generate stuff
+    # that "should" be present for the page to parse better
+    tree = wxr.wtp.parse(
+        page_text,
+        pre_expand=True,
+        additional_expand=ADDITIONAL_EXPAND_TEMPLATES,
+    )
+
+    # print_tree(tree)
+
+    # If this was a normal wiktionary edition, this is where you would split
+    # the page into different language entries; "English", "Gaelic", "Swahili",
+    # etc.
+    # Each section would be a Level 2 node (`NodeKind.LEVEL2`) child of the
+    # page's root node. Wiktionaries do not seem to use LEVEL1 for anything,
+    # although there might be (there definitely is...) an exception somewhere.
+
+    # Some data that is collected is shared among several entries; things like
+    # pronunciation data and etymological data is collected and updated in this
+    # base_data object that can be copied for more specific entries.
+    # Simple English Wiktionary has these sections under Level 3 ("===")
+    # headers, which put them in an awkward hierarchy with the main page and
+    # Part-of-Speech sections, which are higher; they usually appear before
+    # POS sections, but a few pages have a more complex structure. We handle
+    # this, specifically for Simple, by preprocessing the text (above) so that
+    # the page is divided into one or more Level 1 nodes.
+    # If the preprocessor has encountered a structure where you have something
+    # that looks like the start of a section (like "Pronunciation") which comes
+    # after POS data has already started, *and* is followed by a POS section
+    # down the line, that's where we split the page with a Level 1 section.
+    base_data = WordEntry(
+        word=wxr.wtp.title or "ERROR_NO_TITLE",
+        # Simple English wiktionary entries are always for English words
+        lang_code="en",
+        lang="English",
+        pos="ERROR_UNKNOWN_POS",
+    )
+
+    word_datas: list[WordEntry] = []
+
+    flattened_levels = list(tree.find_child_recursively(LEVEL_KIND_FLAGS))
+    headings_of_levels = [
+        clean_node(wxr, None, level.largs[0]).lower()
+        for level in flattened_levels
+    ]
+
+    for i, (heading_title, level) in enumerate(
+        zip(headings_of_levels, flattened_levels)
+    ):
```

Comment:
Could be further simplified, please see my explanation in the next comment. 

```suggestion
    for level_node in tree.find_child_recursively(LEVEL_KIND_FLAGS):
        heading_title = clean_node(wxr, None, level_node.largs).lower()
```

and my suggestion code could have bugs, don't apply them directly...